.changeset/perfect-icons-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"ts-essentials": minor
+---
+
+Deprecate Exact and introduce IsExact<Type,Shape>
```

Comment:
You can use backticks here and can omit type parameters (as it's not as important as in docs)

```suggestion
Deprecate `Exact` and renamed it to `IsExact`
```